app/assets/stylesheets/basic.scss

```diff
@@ -0,0 +1,36 @@
+
+
+.background {
+  background-color: #ADD8E6;
```

Comment:
Color literals like `#ADD8E6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#ADD8E6` should be written as `#add8e6`